tests/layer_tests/tensorflow_tests/test_tf_BiasAdd.py

```diff
@@ -1,156 +1,90 @@
+
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             def create_bias_add_2_consts_net(self, shape, ir_version, use_legacy_frontend, output_type = tf.float32) : tf.compat.v1.reset_default_graph() tf_concat_axis = - 1
+
+#Create the graph and model
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         with tf.compat.v1.Session() as sess : tf_x_shape = shape.copy() tf_y_shape = tf_x_shape[- 1 : ]
+
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 constant_value_x = np.random.randint(- 256, 256, tf_x_shape).astype(output_type.as_numpy_dtype()) x = tf.constant(constant_value_x) constant_value_y = np.random.randint(- 256, 256, tf_y_shape).astype(output_type.as_numpy_dtype()) y = tf.constant(constant_value_y)
```

Comment:
revert changes in tests and simple add test for complex case

Reply:
Hey Roman, my bad. 
I probably ran Clang for the entire local package which resulted in this interestingly. I'll revert.